README.md

```diff
@@ -127,7 +127,8 @@ Important options are
 - `-G Ninja` to use the Ninja backend instead of the default Make backend. Other build backends are available with a similar syntax.
 - `-DCMAKE_INSTALL_PREFIX` is used to provide the install location for the library.
 - `-DCMAKE_MAXIMUM_RANK` the maximum array rank procedures should be generated for.
-  The default is 15 for Fortran 2003 compliant compilers, otherwise 7 for compilers not supporting Fortran 2003 completely yet.
+  The default value is chosen as 4.
```

Comment:
```suggestion
  The default value is chosen as 4, which is also the minimum required rank to compile this project.
```
To delete: line 132<

Reply:
This is mentioned in the next sentence already.

Reply:
Yes, I know. I propose to combine them ;) So the next one should be deleted